internal/worker/firewaller/firewaller.go

```diff
@@ -526,26 +539,26 @@ func (fw *Firewaller) startUnit(ctx context.Context, unit Unit, machineTag names
 	}
 
 	applicationTag := application.Tag()
-	unitTag := unit.Tag()
+	unitName := unit.Name()
```

Comment:
Just an FYI more than anything, the unit.Tag ensures that it conforms to the right unit tag. As long as there is consistency with the unit/port service I'm happy. Otherwise, we should ensure that the port service does infact ensure that the unitName is a valid type.